testing/keybase-client/APKBUILD

```diff
@@ -1,18 +1,14 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=keybase-client
-pkgver=1.0.42
+pkgver=1.0.44
 _ver=${pkgver/_p/-}
 pkgrel=0
 pkgdesc="Keybase command line client"
 url="https://github.com/keybase/client"
 arch="all"
 license="BSD-3-Clause"
-depends=""
```

Comment:
Please don't remove variables set to an empty string, we have no clear policy on this and in the long run someone probably just readds them.

Reply:
Fixed. Thank you for the clarification.